packages/rxjs/readme.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import 'vue-tsx-support/enable-check'
+import Vue from 'vue'
+import { storiesOf } from '@storybook/vue'
+import { defineComponent, ref } from 'vue-demi'
+import { ShowDocs } from '../_docs/showdocs'
+import { useObservable } from './useObservable'
+import { fromEvent } from './from'
+import { of, forkJoin } from 'rxjs'
+import { ajax } from 'rxjs/ajax'
+import { take, mergeMap, concatAll, pluck, map, scan } from 'rxjs/operators'
+
+type Post = {
+  id: number
+  userId: number
+  title: string
+}
+type User = {
+  username: string
+}
+type Comment = {
+  body: string
+}
+type JoinedPost = {
+  username: string
+  comments: number
+  id: string
+}
+
+const Demo = defineComponent({
+  setup() {
+    const BASE_URL = 'https://jsonplaceholder.typicode.com'
+    const button = ref<HTMLButtonElement>(null)
+    const posts = useObservable(fromEvent(button, 'click').pipe(
+      mergeMap(() => ajax.getJSON<Post[]>(`${BASE_URL}/posts`).pipe(
+        concatAll(),
```

Comment:
here i inserted a startWith([]) but was useless

Reply:
Sorry I am not familiar with rxjs, is there anything I can help with?